docs/source/contribute/development/developer_guide/ci.rst

```diff
@@ -4,20 +4,18 @@
 Continuous integration
 ======================
 
-.. _gha: https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions
-.. _precommit: https://pre-commit.ci/
+.. _Github Actions: https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions
+.. _precommit.ci: https://pre-commit.ci/
 
-``skbase`` uses `Github Actions <gha>`_ continuous integration (CI) services
+``skbase`` uses `Github Actions`_ continuous integration (CI) services
 to ensure contributions meet the project's standards. See the sections below to
 see how the project automatically validates code quality, and builds and tests
 your changes.
 
 Code quality checks
 ===================
 
-
-
-``skbase`` uses `pre-commit.ci <precommit>`_ to help maintain the project's
+``skbase`` uses `pre-commit.ci <https://pre-commit.ci/>`_ to help maintain the project's
```

Comment:
would it not be better to have just one link (as it was), in case it changes?

Reply:
So there is something weird going on with the Sphinx linking. I can't get these to work unless I include the URL in-line (though randomly using the reference with spaces in the reference name seemed to be working). I don't really want to investigate why this is going on. We can see if it fixes itself in a future release, but for now this gets the links working.